Data/DcDuty-German.json

```diff
@@ -0,0 +1,1829 @@
+        "738": "Krieger des Lichts",
+        "739": "Krieger des Lichts (extrem)",
+        "74": "Kampf gegen die Dhorme-Chimära",
+        "742": "Diadem",
+        "744": "Traumprüfung - Shiva",
+        "745": "Das Karussell von Lyhe Ghiah",
+        "746": "Matoyas Atelier",
+        "747": "Edens Verheißung - Litanei",
+        "748": "Edens Verheißung - Litanei (episch)",
+        "749": "Edens Verheißung - Umbra",
+        "75": "Kampf gegen die Hydra",
+        "750": "Edens Verheißung - Umbra (episch)",
+        "751": "Edens Verheißung - Anamorphose",
+        "752": "Edens Verheißung - Anamorphose (episch)",
+        "753": "Diadem",
+        "754": "Gogo der Mime",
+        "755": "Triple Triad: Manderville-Turnier",
+        "756": "Triple Triad: Privatturnier",
+        "757": "Traumprüfung - Titan",
+        "758": "Edens Verheißung - Ewigkeit",
+        "759": "Edens Verheißung - Ewigkeit (episch)",
+        "76": "Duell auf der großen Brücke",
+        "760": "Delubrum Reginae",
+        "761": "Delubrum Reginae (episch)",
+        "762": "Smaragdsturm - Entfesselung",
+        "763": "Smaragdsturm - Trauma",
+        "766": "Anfänger-Mahjong (schnelle Partie, gewertet)",
+        "767": "Fortgeschrittenen-Mahjong (schnelle Partie, gewertet)",
+        "768": "4-Spieler-Mahjong (schnelle Partie, Kuitan aktiviert)",
+        "769": "4-Spieler-Mahjong (schnelle Partie, Kuitan deaktiviert)",
+        "77": "Götterdämmerung - Ramuh",
+        "770": "Auf großer Fahrt",
+        "771": "Auf großer Fahrt",
+        "772": "Auf großer Fahrt",
+        "773": "Auf großer Fahrt",
+        "774": "Auf großer Fahrt",
+        "775": "Auf großer Fahrt",
+        "777": "Die Goldene Ebene von Paglth'an",
+        "778": "Zadnor-Hochebene",
+        "78": "Zenit der Götter - Ramuh",
+        "781": "Diamantblitz - Entfesselung",
+        "782": "Diamantblitz - Trauma",
+        "79": "Götterdämmerung - Shiva",
+        "799": "Der Turm, Paradigmenbrecher",
+        "8": "Brüllvolx' Langrast",
+        "80": "Zenit der Götter - Shiva",
+        "81": "Das Drachenhals-Kolosseum",
+        "82": "Jenseits Urths Quelle",
+        "83": "Der Schicksalsweg",
+        "84": "Chrysalis",
+        "85": "Revanche in den Ruinen",
+        "86": "Götterdämmerung - Ravana",
+        "87": "Zenit der Götter - Ravana",
+        "88": "Götterdämmerung - Bismarck",
+        "89": "Zenit der Götter - Bismarck",
+        "9": "Versunkener Tempel von Qarn",
+        "90": "Singularitäts-Reaktor",
+        "91": "Heldenlied von Thordans Fall",
+        "92": "Kristallturm - Das Labyrinth der Alten",
+        "93": "Verschlungene Schatten - T1",
+        "94": "Verschlungene Schatten - T2",
+        "95": "Verschlungene Schatten - T3",
+        "96": "Verschlungene Schatten - T4",
+        "97": "Verschlungene Schatten - T5",
+        "98": "Verschlungene Schatten 2 - T1",
+        "99": "Verschlungene Schatten 2 - T2"
+    },
+    "language": "German",
```

Comment:
```suggestion
    "language": "Deutsch",
```

This changes the word here.
![K-005](https://user-images.githubusercontent.com/4695371/120900047-cd1e9300-c66d-11eb-975d-97be797265d2.png)

Like this
![K-006](https://user-images.githubusercontent.com/4695371/120900054-d7d92800-c66d-11eb-8cf4-dc9d1c187b9f.png)